demos/fleet_provisioning/fleet_provisioning_with_csr/demo_template.json

```
@@ -0,0 +1,52 @@
+ {
+   "Parameters": {
+     "SerialNumber": {
+       "Type": "String"
+     },
+     "AWS::IoT::Certificate::Id": {
+       "Type": "String"
+     }
+   },
+   "Resources": {
+     "certificate": {
+       "Properties": {
+         "CertificateId": {
+           "Ref": "AWS::IoT::Certificate::Id"
+         },
+         "Status": "Active"
+       },
+       "Type": "AWS::IoT::Certificate"
+     },
+     "policy": {
+       "Properties": {
+         "PolicyName": "demo_thing_policy"
```

(Nit) Using `<text>` format can be easier for reader to notice as field that they need to replace
```suggestion
         "PolicyName": "<aws-iot-policy-name>"
```